masked_pixels: [
75, 118,
75, 119,
76, 117,
76, 120,
77, 118,
77, 119,
]
